_data/simple-icons.json

```
@@ -940,6 +940,11 @@
             "hex": "1A86FD",
             "source": "https://buddy.works/about"
         },
+        {
+            "title": "Buefy",
+            "hex": "7957D5",
+            "source": "https://github.com/buefy/buefy"
```

Following [the Contributing Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines), could you update this to:

```suggestion
            "source": "https://github.com/buefy/buefy/blob/a9a724efca0b531e6a64ab734889b00bf4507a9d/static/img/icons/safari-pinned-tab.svg"
```